variables.tf

```diff
@@ -9,6 +9,23 @@ variable "name" {
   }
 }
 
+variable "resource" {
+  description = <<-EOF
+    The identifier of the GCP Resource to monitor.
+
+    The resource can be a project, folder, or organization.
+
+    Examples: "projects/my_project-123", "folders/1234567899", "organizations/34739118321"
+  EOF
+  type        = string
+  default     = "projects/terraflood-345116" # TODO(luke): remove
+
+  validation {
```

Comment:
Nice! Additionally, validate that `length(split("/", var.resource)) == 2` first. I can't remember off hand if these validators run in series and error after the first or collect all errors. Either way, helps avoid obscure errors for an input `resource = "folders"` which is a mistake someone who misunderstood the variable could conceivably make.

Reply:
yeah good point, this will make things clearer